<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title></title>
</head>
<body>

</body>
</html>
<script>
    try {
        document.writeln("开始执行try块语句 ---> ");
        document.writeln("还没有发生例外 ---> ");
        var tt=(prompt("输入一个值：", ""));
        console.log(tt)
        if(tt==123){
            throw new Error('Error description');
        }
    } catch (err) {
        document.writeln("捕捉到例外，开始执行catch块语句 --->");
        console.log(err.name)
        document.writeln("错误名称: " + err.name + " ---> ");
        document.writeln("错误信息: " + err.message + " ---> ");
    } finally {
        document.writeln("开始执行finally块语句")
    }
</script>
　　